src/components/screens/HomeScreen.js

```diff
@@ -37,61 +46,75 @@ export const StyledCounter = styled.h1`
 
 export default class HomeScreen extends Component {
   state = {
-    isBookmarked: this.loadFavorites(),
-    iconIsDefault: true,
-    search: ''
+    isFestBookmarked: this.loadFavoriteFests(),
```

Comment:
The key "isFestBookmarked" is odd. I would expect a boolean for a single festival here.
Rename to "favoriteFests" or similar.